acceptance/tests/yaml_backend/00-setup.rb

```diff
@@ -1,6 +1,8 @@
 test_name "Hiera setup for YAML backend"
 
 agents.each do |agent|
+  puppetcodedir = agent.puppet()[:codedir]
+  hieradatadir = "#{puppetcodedir}/hieradata"
   apply_manifest_on agent, <<-PP
 file { '/etc/puppetlabs':
```

Comment:
I don't think we ever tested hiera on windows, I filed a ticket a long time ago to start testing it